.vscode/launch.json

```diff
@@ -37,10 +37,10 @@
       "args": ["${workspaceFolder}/sample/app.js"],
 
       "env": {
-          "DYLD_INSERT_LIBRARIES": "${workspaceFolder}/target/debug/libmirrord.dylib",
-          "LD_PRELOAD": "${workspaceFolder}/target/debug/libmirrord.so",
-          "RUST_LOG": "trace",
-          "MIRRORD_AGENT_IMPERSONATED_POD_NAME": "nginx-deployment-66b6c48dd5-xg4rq",
+          "DYLD_INSERT_LIBRARIES": "${workspaceFolder}/target/debug/libmirrord_layer.dylib",
+          "LD_PRELOAD": "${workspaceFolder}/target/debug/libmirrord_layer.so",
+          "RUST_LOG": "warn,mirrord=trace",
+          "MIRRORD_AGENT_IMPERSONATED_POD_NAME": "nginx-deployment-66b6c48dd5-c47lc",
```

Comment:
We should probably just leave this empty, since the name can change

Reply:
Done `"MIRRORD_AGENT_IMPERSONATED_POD_NAME": ""`.